infra/growth/src/resources/eventLogger.js

```diff
@@ -0,0 +1,56 @@
+const Web3 = require('web3')
+
+const { GrowthEvent } = require('@origin/growth-shared/src/resources/event')
+const { GrowthEventTypes } = require('@origin/growth-shared/src/enums')
+
+const logger = require('../logger')
+
+/**
+ * Endpoint for logging entries in the growth_event table.
+ * For now we only support logging events of the BrowserExtensionInstalled type.
+ * Expected paramaters:
+ *  - type: "BrowserExtensionInstalled"
+ *  - eth_address: eth address the user used when signing up for origin rewwards.
+ *  - uid: unique browser extension id
+ * @param req
+ * @param res
+ * @returns {Promise<*>}
+ */
+async function logEvent(req, res) {
+  // Check presence and validity of the type parameter.
+  const type = req.query.type
+  if (type !== 'BrowserExtensionInstalled') {
+    return res.status(400).end('Invalid event type')
+  }
+
+  // Check presence and validity of the eth_address parameter.
+  const ethAddress = req.query.eth_address
+  if (!ethAddress) {
+    return res.status(400).end('Missing eth_address')
+  }
+  if (!Web3.utils.isAddress(ethAddress)) {
+    return res.status(400).end('Invalid eth_address')
+  }
+
+  // Check presence and validity of the uid parameter
+  // TODO(franck): check validity of the uid.
+  const uid = req.query.uid
```

Comment:
Just realised this uses query strings instead of the POST body, is that a growth standard?

Reply:
Not at all. If easier for the client to pass args in the body, I'm happy to change this endpoint. Lmk...

Reply:
That'd be great, it might be easier. If I pass everything in the query string express errors with a body required error so body is probably mandatory.

FYI I've submitted an initial build of the extension with this growth call with the parameters in the body.

Reply:
No problem. I'll make this change first thing on Monday...